lib/age.rb

```diff
@@ -0,0 +1,15 @@
+class Age
+  def self.from_date(date_of_birth)
+    now = Time.now.utc.to_date
+
+    leap_year_correction =
```

Comment:
Isn't this mainly to subtract one year from birth dates that are in the future of the current year? If so then the variable name is a bit misleading (although it's correct that this version also works for leap years).